resources/js/components/Common/FormButton.vue

```diff
@@ -1,13 +1,14 @@
 <script setup lang="ts">
-
 import { router } from '@inertiajs/vue3'
 import {ref} from 'vue'
+import Button from '@/components/Common/Button.vue'
 
 const props = withDefaults(defineProps<{
   small?: boolean
   disabled?: boolean
   text?: boolean
   class?: string
+  buttonClass?: string
```

Comment:
Look at this

![image](https://github.com/user-attachments/assets/1233a285-3e73-420e-8208-eedb1796c54f)

Reply:
fixed